CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 0.30.0 **UNRELEASED**
 
+- Add support for Dokka 2.0.0-Beta
+    - Supports `org.jetbrains.dokka.experimental.gradle.pluginMode=V2Enabled`
+    - When `pluginMode` is set to `V2EnabledWithHelpers` the old v1 tasks are used
```

Comment:
FYI: in case of `V2EnabledWithHelpers` we do only create stub tasks which matches old types - they will fail to generate documentation.
Still, it should not really matter, as `V2EnabledWithHelpers` should only be used during migration and should not be committed or used afterwards